src/pages/Exhibitions/Role/Role.tsx

```diff
@@ -0,0 +1,13 @@
+// Libraries
+import React from "react";
+import RoleProps from "./models/RoleProps";
+// Styles
+import "./style.css";
+
+const Role: React.FC<RoleProps> = (props: RoleProps) => (
+  <div>
+    {/* <h3>{props.role}</h3> */}
```

Comment:
@sophiewongsp It's ok to show the role here. You just need to change your syntax. You can refer to [`ProjectRowText`](https://github.com/niji-co/Fusion-Web/blob/master/src/pages/Project/Row/ProjectRowText.tsx) for how I did it. (There are other ways too so feel free to use them too)